lib/util.js

```diff
@@ -471,3 +472,22 @@ exports.deprecate = (obj, prop, value, oldPath, newPath) => {
     },
   });
 };
+
+
+/**
+ * Gets random IPv6 Address from a block
+ *
+ * @param {string} block
+ * @returns {string}
+ */
+exports.getRandomIPv6 = block => {
+  let subnet = parseInt(block.split('/')[1]);
+  block = block.split('/')[0];
+  if (!subnet || subnet > 128 || subnet < 24) throw new Error('Invalid IPv6 subnet');
```

Comment:
```suggestion
  if (!subnet || subnet > 128 || subnet < 24) throw Error('Invalid IPv6 subnet');
```

Reply:
not sure that this is covered in the tests either. wrap it in `{}` to see in it's caught. i'll make sure to add an eslint rule about this in the future.